tests/integrations/test_convergence.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from tests.integrations import config as conf
+from tests.integrations import experiment as exp
+
+
+@pytest.mark.nightly  # type: ignore
+def test_cifar10_pytorch_accuracy() -> None:
+    config = conf.load_config(conf.official_examples_path("cifar10_cnn_pytorch/const.yaml"))
+    config = conf.set_max_steps(config, 2)
```

Comment:
issue: don't think we want to do this.. we should be directly testing on `conf.official_examples_path("cifar10_cnn_pytorch/const.yaml")` right?

Reply:
Good catch -- accidentally left this step modification from testing. removed.